test/validator.test.ts

```diff
@@ -21,6 +21,52 @@ import {
 } from '../src/utils/validator.js'
 import type { Intention } from '../src/types/core.js'
 
+// --- MOCK DATA FOR TESTS ---
+
+const mockValidIntention: Intention = {
+	action: 'buy 1000 USDC of WETH',
```

Comment:
This `action` should be phrased in an explicitly provable way, like `Swap 1,000 USDC for 0.3 WETH with .0111 WETH in fees`